src/Build/BackEnd/Node/OutOfProcNode.cs

```diff
@@ -515,6 +513,11 @@ private NodeEngineShutdownReason HandleShutdown(out Exception exception)
                 CleanupCaches();
             }
 
+            if (_debugCommunications)
+            {
+                CommunicationsUtilities.Trace("Shut down complete.", Process.GetCurrentProcess().Id);
```

Comment:
For both of these, I think the order of your arguments is reversed. See [here](https://github.com/microsoft/msbuild/blob/master/src/Shared/CommunicationsUtilities.cs#L604)

Reply:
I'm using the overload [above that one](https://github.com/microsoft/msbuild/blob/master/src/Shared/CommunicationsUtilities.cs#L596). I actually mistakenly left the second argument in there. We don't need to pass the current process ID so I'll get rid of that.